libraries/AP_BattMonitor/AP_BattMonitor.cpp

```diff
@@ -841,24 +841,25 @@ void AP_BattMonitor::check_failsafes(void)
 // return true if any battery is pushing too much power
 bool AP_BattMonitor::overpower_detected() const
 {
-    bool result = false;
+#if AP_BATTERY_WATT_MAX_ENABLED && APM_BUILD_TYPE(APM_BUILD_ArduPlane)
```

Comment:
Just generally, if you see #ifs excluding the entire body of a function it's worth having a look at the callers and seeing if it's convenient to remove the entire function rather than just the body.  Often the additional `#if` is useful as it points out there are mavro-IFs controlling behaviour, not just C++-ifs....


... in this case we'd save several hundred bytes on Plane, I think....